Subject: domcapabilities: Get video devices
From: Fabiano Fidêncio fidencio@redhat.com Thu Oct 3 10:44:52 2019 +0200
Date: Fri Oct 4 11:15:09 2019 -0400:
Git: 1547e16d62736c861a0e4dadeb86ef372d2698bd

domcapabilities already handles disk and hostdev. Let's add support for
getting video devices as well.

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

diff --git a/virtinst/domcapabilities.py b/virtinst/domcapabilities.py
index f11f18aa..9cc27bc2 100644
--- a/virtinst/domcapabilities.py
+++ b/virtinst/domcapabilities.py
@@ -95,6 +95,7 @@ class _Devices(_CapsBlock):
     XML_NAME = "devices"
     hostdev = XMLChildProperty(_make_capsblock("hostdev"), is_single=True)
     disk = XMLChildProperty(_make_capsblock("disk"), is_single=True)
+    video = XMLChildProperty(_make_capsblock("video"), is_single=True)
 
 
 class _Features(_CapsBlock):
